.github/workflows/application-pull-request.yml

```diff
@@ -24,6 +24,8 @@ concurrency:
 jobs:
   test:
     uses: ./.github/workflows/test.yml
+    secrets:
+      SAM_APP_VALIDATE_ROLE_ARN: ${{ secrets.SAM_APP_VALIDATE_ROLE_ARN }}
```

Comment:
Is this really that sensitive? Or does it differ by env?

Reply:
This is from the guide from DI, so I have kept it sensitive for their benefit